projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -58,22 +57,42 @@ import { MultiSelectJustify } from './multi-select-justify';
         <ht-popover-content>
           <div class="multi-select-content" [ngStyle]="{ 'min-width.px': triggerContainer.offsetWidth }">
             <ng-container *ngIf="this.enableSearch">
-              <ht-search-box
-                class="search-bar"
-                (valueChange)="this.searchOptions($event)"
-                displayMode="${SearchBoxDisplayMode.NoBorder}"
-              ></ht-search-box>
+              <ng-container *ngIf="this.allOptions$ | async as allOptions">
+                <ng-container *ngIf="allOptions.length > 5">
```

Comment:
Hmm so if enable search is disabled, these will never show. If true, it shows when at least 5 options? I thought you were describing that the > 5 rule was the default if enableSearch is unset. Anyway we can go with it and address as use cases pop up.